scalafmt-interfaces/src/main/java/org/scalafmt/interfaces/ScalafmtSession.java

```diff
@@ -0,0 +1,27 @@
+package org.scalafmt.interfaces;
+
+import java.nio.file.Path;
+
+/**
+ * A session based on a fixed configuration.
+ */
+public interface ScalafmtSession {
```

Comment:
It is in interface module because `ScalafmtSession` should be supported in another integrations (sbt plugin for example)?